### ISSUE TITLE
unknown-target 'minion sync status' 

### ISSUE THREAD

**Issue by kerncore** [tinyclues/saltpad#73]
Hi, 

When try send highstate test=true (Sync minion status)
Getting back correctly parsed highstate output but with 'Target': 'unknown-target'. 
Following that minion sync-status can't be checked.

The full trace output from salt-api is attached below:

[TRACE   ] get_event() received = {'tag': 'salt/event/new_client', 'data': {'_stamp': '2015-07-12T20:14:48.674521'}}
[TRACE   ] get_event() received = {'tag': 'salt/run/20150712201448689394/ret', 'data': {'jid': '20150712201448689394', 'return': {'Function': 'state.highstate', 'jid': '20150712201448253266', 'Target': 'qa-dev', 'Target-type': 'glob', 'User': 'kerncore', 'StartTime': '2015, Jul 12 20:14:48.253266', 'Arguments': [{'test': True, '__kwarg__': True}], 'Minions': ['qa-dev'], 'Result': {}}, 'success': True, '_stamp': '2015-07-12T20:14:49.093392', 'user': 'kerncore', 'fun': 'runner.jobs.list_job'}}
[INFO    ] 173.255.115.105 - - [12/Jul/2015:20:14:49] "GET /jobs/20150712201448253266 HTTP/1.1" 200 285 "" "python-requests/2.7.0 CPython/2.7.9 Linux/3.16-0.bpo.3-amd64"
[DEBUG   ] MasterEvent PUB socket URI: ipc:///var/run/salt/master/master_event_pub.ipc
[DEBUG   ] MasterEvent PULL socket URI: ipc:///var/run/salt/master/master_event_pull.ipc
[DEBUG   ] Sending event - data = {'_stamp': '2015-07-12T20:14:54.389242'}
[TRACE   ] get_event() received = {'tag': 'salt/event/new_client', 'data': {'_stamp': '2015-07-12T20:14:54.389242'}}
[TRACE   ] get_event() received = {'tag': 'salt/run/20150712201454390849/ret', 'data': {'jid': '20150712201454390849', 'return': {'outputter': 'highstate', 'data': {'qa-dev': {'sysctl_|-vm.overcommit_memory_|-vm.overcommit_memory_|-present': {'comment': 'Sysctl value vm.overcommit_memory = 1 is already set', 'name': 'vm.overcommit_memory', 'start_time': '20:14:49.853478', 'result': True, 'duration': '23.278 ms', '__run_num__': 0, 'changes': {}}}}}, 'success': True, '_stamp': '2015-07-12T20:14:54.539239', 'user': 'kerncore', 'fun': 'runner.jobs.lookup_jid'}}
[DEBUG   ] MasterEvent PUB socket URI: ipc:///var/run/salt/master/master_event_pub.ipc
[DEBUG   ] MasterEvent PULL socket URI: ipc:///var/run/salt/master/master_event_pull.ipc
[DEBUG   ] Sending event - data = {'_stamp': '2015-07-12T20:14:54.566350'}
[TRACE   ] get_event() received = {'tag': 'salt/event/new_client', 'data': {'_stamp': '2015-07-12T20:14:54.566350'}}
[TRACE   ] get_event() received = {'tag': 'salt/run/20150712201454574005/ret', 'data': {'jid': '20150712201454574005', 'return': {'Function': 'state.highstate', 'jid': '20150712201448253266', 'Target': 'unknown-target', 'Target-type': [], 'User': 'root', 'StartTime': '2015, Jul 12 20:14:48.253266', 'Arguments': [], 'Minions': ['qa-dev'], 'Result': {'qa-dev': {'return': {'sysctl_|-vm.overcommit_memory_|-vm.overcommit_memory_|-present': {'comment': 'Sysctl value vm.overcommit_memory = 1 is already set', 'name': 'vm.overcommit_memory', 'start_time': '20:14:49.853478', 'result': True, 'duration': 23.278, '__run_num__': 0, 'changes': {}}}, 'out': 'highstate'}}}, 'success': True, '_stamp': '2015-07-12T20:14:54.832761', 'user': 'kerncore', 'fun': 'runner.jobs.list_job'}}
[INFO    ] 173.255.115.105 - - [12/Jul/2015:20:14:54] "GET /jobs/20150712201448253266 HTTP/1.1" 200 883 "" "python-requests/2.7.0 CPython/2.7.9 Linux/3.16-0.bpo.3-amd64"

**Comment by kerncore** [comment]
Found it )
https://github.com/saltstack/salt/issues/21496

**Comment by Lothiraldan** [comment]
Yeah, it's an issue on the salt side, unfortunately I could not fix it on my side, you could add a comment if this issue is annoying

**Comment by kerncore** [comment]
Work in v2015.5.3

**Comment by pcn** [comment]
I don't use saltpad, but I can confirm that this appears to be fixed in the netapi in 2015.5.3 as well.  You can probably close this issue.
